### PR TITLE
Migration small fixes

### PR DESCRIPTION
**Pull request by alexandraorth**
+ User can add macros
+ Help message prints once
+ Testing allows for system exit

### CODE REVIEW

**Comment by alexandraorth** [comment]
please retest this please

**Comment by alexandraorth** [comment]
please retest this please

**Comment by alexandraorth** [comment]
please retest this please

File: mindmaps-graql/src/main/java/io/mindmaps/graql/internal/template/TemplateParser.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 public class TemplateParser {
 
-    private static Map<String, Macro<Object>> macros;
+    private final Map<String, Macro<Object>> macros = new HashMap<>();
```
Comment (fppt) [inline mindmaps-graql/src/main/java/io/mindmaps/graql/internal/template/TemplateParser.java]: why not static anymore ? was this an error ?

**Pull request merged**